data/campaigns/The_Hammer_of_Thursagan/scenarios/12_The_Underlevels.cfg

```diff
@@ -246,7 +247,52 @@
         {VARIABLE rune_one 0}
         {VARIABLE rune_two 0}
     [/event]
-
+    
+    [event]
+        name=recruit
+        first_time_only=no
+        [filter]
+            side=1
+        [/filter]
+        
+        [object]
+            silent=yes
+            take_only_once=no
```

Comment:
I think this key serves no purpose.
But you can add duration=scenario

Reply:
It does do something but I think it's basically redundant in a one shot event.